src/main/java/repo/impl/PostgresFormSessionRepo.java

```diff
@@ -73,10 +77,12 @@
 
         if(sessionCount > 0){
             String query = replaceTableName("UPDATE %s SET instanceXml = ?, sessionData = ?, " +
-                    "sequenceId = ?, currentIndex = ? WHERE id = ?");
+                    "sequenceId = ?, currentIndex = ?, postUrl = ?, restoreXml = ? WHERE id = ?");
```

Comment:
sweet. just flagging now we could potentially have 3 different types of restores in a db. one from a restoreAs, a regular restore, and a old session one. the old session one is probably the one that would need to be namespaced the most if possible. i can imagine someone opening an old session in one screen and then in another filling out a form